galgebra/printer.py

```diff
@@ -1022,7 +1024,7 @@ def Print_Function():
     tmp_str = inspect.getsource(inspect.currentframe().f_back)
     if GaLatexPrinter.latex_flg:
         #print '#Code for '+fct_name
-        print(r'##\\begin{lstlisting}[language=Python,showspaces=false,' + \
+        print(r'##\\begin{lstlisting}[language=Python,showspaces=false,'
```

Comment:
Why do we need `\\` here?

Reply:
We don't, and this is actually a bug. It's out of scope for a flake 8 patch though

Reply:
I can fix this in a follow-up PR - it's actually wrong in multiple places.